src/login/Login.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import "./Login.css";
+import { Redirect } from "react-router-dom";
+import "../myspace/MySpace";
+
+
+export default function Login() {
+    const [username, setUsername] = useState(true);
```

Comment:
I see, the value of username, password, accessToken and accessExpiry is never read. Then why are you setting it? you don't need it.

Reply:
does that mean I should remove onChange on each field attributes? Or what should I replace it with? How will the field changes get recorded?

<img width="543" alt="Screen Shot 2020-06-25 at 9 46 35 am" src="https://user-images.githubusercontent.com/29667122/85638320-efbbd400-b6c8-11ea-8033-caf371f38cf6.png">

Reply:
Done, remove onChange